src/store/modules/swap/ethBancor.ts

```diff
@@ -563,6 +563,64 @@ const determineConverterType = (
   throw new Error("Failed to determine the converter type");
 };
 
+const getHistoricFees = async (
+  id: string,
+  converterAddress: string,
+  network: EthNetworks
+): Promise<PreviousPoolFee[]> => {
+  const w3 = getWeb3(network);
+  const contract = buildV28ConverterContract(converterAddress, w3);
+  const { blockHoursAgo, currentBlock } = await blockNumberHoursAgo(
```

Comment:
Pass the blocks as a param to this ```getHistoricFees``` function, need not call on a per converter basis.

Reply:
done